### ISSUE TITLE
No such file or directory: sphinxcontrib_httpexample-0.4.2-py2.7.egg/sphinxcontrib/httpexample/static/sphinxcontrib-httpexample.css

### ISSUE THREAD

**Issue by tisto** [collective/sphinxcontrib-httpexample#3]
Running bin/sphinxbuilder on the plone.restapi repo leads to the following traceback:

```
Copying osexample stylesheet/javascript... done

Exception occurred:
  File "/usr/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: '/home/timo/.buildout/eggs/sphinxcontrib_httpexample-0.5.1-py2.7.egg/sphinxcontrib/httpexample/static/sphinxcontrib-httpexample.css'
The full traceback has been saved in /tmp/sphinx-err-IjaX9j.log, if you want to report the issue to the developers.
```

**Comment by tisto** [comment]
OS: Ubuntu 14.04, Python 2.7. It seems the static folder is indeed missing from the release. Maybe a MANIFEST.ini issue?

**Comment by csenger** [comment]
I fixed the packaging.

@datakurre Can you cut a new release or add me to pypi?

**Comment by datakurre** [comment]
@csenger, @tisto Thanks. I added @csenger to PyPI maintainers of the package.

**Comment by csenger** [comment]
Fixed in 0.5.2.
